crate/operator/expand_volume.py

```diff
@@ -184,3 +187,104 @@ async def handle(  # type: ignore
             ),
             WebhookStatus.SUCCESS,
         )
+
+
+class StartClusterSubHandler(StateBasedSubHandler):
```

Comment:
I think we moved `StartClusterSubHandler` and `SuspendClusterSubHandler` intentionally to the generic file back then to be able to reuse it from different operations.

https://github.com/crate/crate-operator/pull/330#pullrequestreview-914265413

> Do you think it might be useful to re-structure things a bit and move these generic handlers into a separate file? As they have nothing to do with expanding volumes per se...

Reply:
I had to move them due to a circular dependency, but will try to overcome it and keep them where they were!